src/main/kotlin/no/nav/familie/ef/sak/api/gui/VurderingController.kt

```diff
@@ -28,4 +32,9 @@ class VurderingController(val vurderingService: VurderingService) {
         return Ressurs.success(vurderingService.hentInngangsvilkår(behandlingId))
     }
 
+    @PostMapping("/{behandlingId}/inngangsvilkar/validering")
+    fun validerInngangsvilkår(@BehandlingConstraint @PathVariable behandlingId: UUID): Ressurs<UUID> {
+        val behandling = behandlingService.hentBehandling(behandlingId)
+        return Ressurs.success(stegService.håndterInngangsvilkår(behandling).id)
```

Comment:
Føler ikke att validering er helt riktig ordbruk her.
Føles som att vi egentligen ønsker å gå videre til neste steg. Hvis det feiler burde vi returnere en liste med  når man egentligen kaller på håndterInngangsvilkår og går videre til neste steg? 

Det er veldig mye som skjer i `håndterSteg` før man kaster selve valideringsfeilet, burde man kanskje kaste det før man gjør allt som skjer der?

Reply:
Hmm tror ikke jeg forstår hva du mener. Valideringen er det som kreves for å kunne gå videre til neste steg. Hvis du får en ok tilbake fra endepunktet så kan du gå videre i frontend, hvis du får feil må frontend-feilmeldingen som sendes med vises. I feilmeldingen står det hva som ikke har validert. Jeg synes det er et bra integrasjonsmønster, vet ikke helt hva du mener med "returnere en liste med når man egentligen kaller på håndterInngangsvilkår".

Reply:
For meg er validering og gå videre til neste steg 2 ulike typer av hendelser, validering skal ikke endre state

Reply:
Vi kan godt kalle endepunktet noe annet - feks `/fullfor`. Men vi kommer ikke unna å gjøre validering her før vi setter behandlingen til neste steg. Rammeverket er jo lagt opp sånn at hvis valideringen feiler, blir behandlingen stående på samme steg, ellers blir steget oppdatert. Hvis du mener det er et dårlig mønster så tenker jeg at det er rammeverket som bør endres, eller at vi bør ta i bruk et annet rammeverk 🤷‍♀️

Reply:
Endret navnet på endepunktet: dba7f5f